demo/demo.js

```diff
@@ -126,60 +125,138 @@ ethSignMsg.addEventListener("click", async () => {
     }
 });
 
+// Display single-sig BTC address on device for verification
 document.getElementById("btcAddressSimple").addEventListener("click", async () => {
-    await device.bitbox02API.btcDisplayAddressSimple(
-        api.firmware.messages.BTCCoin.BTC,
+    await device.api.btcDisplayAddressSimple(
```

Comment:
The demo file is getting quite large and hard to navigate.
Sooner rather than later we should refactor all the button functionality into a separate file I'd say.